File: src/main/java/com/fractalwrench/bpp/internal/ast/StringLiteralNode.java
package com.fractalwrench.bpp.internal.ast;

import org.objectweb.asm.Opcodes;


public class StringLiteralNode extends AstNode implements Opcodes {

    private String input;

    public StringLiteralNode(String input, AstNode left, AstNode right) {
        super(left, right);
        this.input = input;
    }

    @Override
    public void generate(jdk.internal.org.objectweb.asm.MethodVisitor methodVisitor) {

    }
}

File: src/main/java/com/fractalwrench/bpp/internal/ast/StringLiteralNode.kt
package com.fractalwrench.bpp.internal.ast

import org.objectweb.asm.Opcodes


class StringLiteralNode(private val input: String, left: AstNode?, right: AstNode?) : AstNode(left, right), Opcodes {

    override fun generate(methodVisitor: jdk.internal.org.objectweb.asm.MethodVisitor) {

    }
}
